src/main/java/reiff/finalproject/RijksSearchFrame.java

```diff
@@ -0,0 +1,134 @@
+package reiff.finalproject;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.net.URL;
+
+import java.io.IOException;
+import javax.imageio.ImageIO;
+
+import com.andrewoid.ApiKey;
+import io.reactivex.rxjava3.core.Single;
+
+public class RijksSearchFrame extends JFrame {
+    private final JTextField searchField;
+    private final JPanel resultsPanel;
+    private int currentPage = 1;
+    private final RijksMuseumService service;
+    private final ApiKey key;
+
+    public RijksSearchFrame() {
+        setTitle("Rijksmuseum Art Search");
+        setSize(800, 600);
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        setLayout(new BorderLayout());
+
+        JPanel topPanel = new JPanel();
+        topPanel.setLayout(new BorderLayout());
+
+        JPanel searchPanel = new JPanel();
+        searchPanel.setLayout(new BorderLayout());
+        searchField = new JTextField();
+        JButton searchButton = new JButton("Search");
+        JButton prevButton = new JButton("Previous Page");
+        JButton nextButton = new JButton("Next Page");
+        searchPanel.add(prevButton, BorderLayout.WEST);
+        searchPanel.add(searchField, BorderLayout.CENTER);
+        searchPanel.add(nextButton, BorderLayout.EAST);
+        searchPanel.add(searchButton, BorderLayout.SOUTH);
```

Comment:
Update your screenshot, but I'm assuming this looks pretty bad.
Instead why don't you have it search when the user presses enter and get rid of the button